lizrd/core/init.py

```diff
@@ -0,0 +1,34 @@
+import torch
+from torch.nn.init import trunc_normal_
+
+
+def get_init_weight(shape, fan_in, init_type="uniform", scale=1.0, dtype=torch.float32):
+    if init_type == "uniform":
+        return init_uniform(shape=shape, fan_in=fan_in, gain=scale, dtype=dtype)
+    elif init_type == "truncated_normal":
+        return init_truncated_normal(
+            shape=shape, fan_in=fan_in, scale=scale, dtype=dtype
+        )
+    else:
+        raise ValueError(f"Unknown init_type: {init_type}")
+
+
+def init_uniform(shape, fan_in, gain=1.0, dtype=torch.float32):
+    range_ = gain * (3 / fan_in) ** 0.5
```

Comment:
Is this 3 important? Why 3 anyway?

Reply:
Please link to the torch PR that justifies this

Reply:
also, do we care about relu in anyway? If we use relu after a linear projection we should, I think, multiply the scale by a factor of two, maybe? altho ignoring it completely sounds fine as well

Reply:
<jk, unless, but joking, but maybe we should> We can also replace relu with ABSOLUTE VALUE for maximum efficiency and to keep the magnitude constant (altho sadly it won't allow for in-place operations) <\jk, unless, but joking, but maybe we should>

Reply:
I very much support this notion

Reply:
After discussion we leave it as is